R/input.R

```diff
@@ -99,15 +155,15 @@ uitextinput <- function(name, value = "", type = "text", placeholder = NULL, att
 #' # Text input
 #' uiinput(
 #'   tags$label("Numeric Input"),
-#'   uinumericinput("ex", 10)
+#'   numeric_input("ex", 10)
 #' )
 #'
 #' @export
-uinumericinput <- function(name, value, min = NA, max = NA, step = NA,
+numeric_input <- function(input_id, value, min = NA, max = NA, step = NA,
```

Comment:
`text_input` has a label argument. Should we make this consistent and add lebel here as well, or remove from text?